Source/Core/Core/Boot/Boot.cpp

```diff
@@ -61,6 +61,7 @@ void CBoot::Load_FST(bool _bIsWii)
 		shift = 2;
 
 	u32 fst_offset, fst_size, max_fst_size;
+	fst_offset = fst_size = max_fst_size = 0;
```

Comment:
Give each variable declaration a line of its own instead:

    u32 fst_offset = 0;
    u32 fst_size = 0;
    u32 max_fst_size = 0;